tests/component/test_watchdog.py

```diff
@@ -0,0 +1,81 @@
+from nidaqmx import Task
+from nidaqmx.constants import WatchdogAOExpirState, WatchdogCOExpirState
+from nidaqmx.system import Device
+from nidaqmx.system.watchdog import AOExpirationState, COExpirationState
+
+
+def test___call_cfg_watchdog_ao_expir_states___no_error(
```

Comment:
Please omit "call". Using a function name for the "when" clause implicitly means to call that function.

Reply:
Removed from all tests that had it in this file.